docs/rules-and-regulations.md

```diff
@@ -17,204 +16,101 @@ The Organization shall not arbitrarily discriminate on the basis of race, nation
+### Section 2.05. Membership Downgrades. 
 
+A membership can also be downgraded, either by request of the member or a cessation of activity required by their current membership level.
 
-### Section 2.07. Membership Voting Rights.
- 
-Each membership level has separate and distinct voting rights as outlined below:
+### Section 2.06. Membership Inactive status.
 
-1. **Community member:** Community members hold limited voting rights and are able to vote on Community issues identified by Partner Members. 
-
-2. **Contributor:** Contributors are limited voting members and are able to vote on  Contributor issues identified by Partner Members. 
-
-3. **Partner Memberships:** Partners are full voting members.
-
-In order for a vote by the members to be valid a quorum of the members of such voting member class must have voted.  A quorum shall consist of at least 75% of the members or voting member class.
- 
-### Section 2.08. Voluntary Withdrawal.
+A membership can be moved to "Inactive" status, either by request of the member, or due to a lack of involvement for a period of X months.
  
-All members shall have the right to resign from the Organization and terminate his or her membership by filing with the Organization written notice of resignation. The resignation shall become effective immediately without any action on the part of the Organization.  Upon resignation, such member shall be entitled to receive all compensation and other monies earned up to the date of the written notice  of resignation.
+### Section 2.07. Fees.
  
-### Section 2.09. Expulsion.
+**Partner Fees.** Current dues for the Partner Members are as follows: $550 paid annually.
  
-A member may, for failure to comply with these Rules and Regulations, standards, or any written code of conduct of the Organization, or for any other reason, be expelled from the Organization by resolution adopted by a two-thirds (2/3) vote of all Partner Members of the Organization. Expulsion shall become effective immediately unless the Organization shall, in the resolution, fix a future time. On expulsion, the name of the member expelled shall be stricken from the register of the Organization and all of his or her voting and fiduciary rights shall cease.  After expulsion, no membership fees will be billed to such member.  However, such member shall be entitled to receive all compensation and other monies earned up to the date of such expulsion.   
+### Section 2.08. Voluntary Withdrawal.
  
+All members shall have the right to resign from the Organization and terminate his or her membership by filing with the Organization written notice of resignation. The resignation shall become effective immediately without any action on the part of the Organization. Upon resignation, such member shall be entitled to receive all compensation and other monies earned up to the date of the written notice of resignation.
```

Comment:
Leaving a comment for now, will attempt an edit next.  I'm still familiarizing myself with GitHub.

We should change "his or hers" to theirs.  We are referring to members in the plural, and we do not need to add gender.